morgoth/base/static/js/components/Page.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes as pt } from 'react';
+
+import AppBar from 'material-ui/AppBar';
+import { blueGrey400, blueGrey500, blueGrey700, orange800 } from 'material-ui/styles/colors';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+
+import Breadcrumbs from './Breadcrumbs.jsx';
+
+
+const muiTheme = getMuiTheme({
+  palette: {
+    primary1Color: blueGrey500,
+    primary2Color: blueGrey700,
+    primary3Color: blueGrey400,
+    accent1Color: orange800,
+  },
+});
+
+const Page = props => {
+  const { routes, children } = props;
```

Comment:
I think this pattern is better as `const Page = ({ routes, children }) => {`.

I'm also not a huge fan of `const Foo = () => {}` vs `function Foo() {}`, but I have no good reasons for that.